views/default/forms/admin/menu/save.php

```diff
@@ -7,11 +7,7 @@
 $num_featured_items = 6;
 
 // get site menu items
-$menu = elgg_get_config('menus');
-$menu = $menu['site'];
-$builder = new ElggMenuBuilder($menu);
-$menu = $builder->getMenu('name');
-$menu_items = $menu['default'];
+$menu_items = elgg_get_menu('site')->getMenuItems('name', 'default');
```

Comment:
I'm cool with adding hooks here as long as we do some testing. If the form affects a menu item that was added via the prepare hook, what happens?